cmd/note/.env

```
@@ -11,6 +11,9 @@ NOTOPIA_NOTE_KAFKA_BROKERS=localhost:19092
 NOTOPIA_NOTE_KAFKA_CONSUMER_GROUP=note-service
 NOTOPIA_NOTE_REDIS_ADDR=localhost:6373
 NOTOPIA_NOTE_SERVICES_AUTHORIZATION_URL=http://localhost:18089
+NOTOPIA_NOTE_AUTHENTIK_HOST=http://authentik.notopia.localhost
+NOTOPIA_NOTE_AUTHENTIK_URL=/api/v3
+NOTOPIA_NOTE_AUTHENTIK_TOKEN=notopiauit
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NOTOPIA_NOTE_AUTHENTIK_TOKEN key should go before the NOTOPIA_NOTE_AUTHENTIK_URL key

(UnorderedKey)